docs/guides/setup-channel.mdx

```diff
@@ -50,24 +50,24 @@ Available plans:
 
 1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account).
 
-1. <span id="save-account"></span>Authenticate to the service by calling `QiskitRuntimeService` with your IBM Quantum API key and CRN:
+1. <span id="save-account"></span>Authenticate to the service by calling `QiskitRuntimeService` with your IBM Quantum API token and CRN. [Set up an environment variable](#securely-manage) rather than copying and pasting your API token into your code.
 
    ```python
    from qiskit_ibm_runtime import QiskitRuntimeService
 
-   service = QiskitRuntimeService(channel="ibm_quantum", token="<MY_IBM_QUANTUM_TOKEN>")
+   service = QiskitRuntimeService(channel="ibm_quantum", token=<your-environment-variable>)
```

Comment:
If we're telling them to set it as IQP_API_TOKEN, why not just use that instead of <your-environment-variable>?

Reply:
I tried this but got an error"...IBMNotAuthorizedError: '401 Client Error: Unauthorized for url: https://auth.quantum-computing.ibm.com/api/users/loginWithToken. Login failed., Error code: 3446.'" 

Maybe I set something wrong?  I found instructions to update .bashrc and added my environment variable, then restarted bash and used it to start jupyter lab.  Then I put this code in a notebook: 

```
from qiskit_ibm_runtime import QiskitRuntimeService
 
service = QiskitRuntimeService(channel="ibm_quantum", token=("IQP_API_TOKEN"))
```

Reply:
right now you are setting the token as the hardcoded string "IQP_API_TOKEN". It would need to be `os.environ.pop("IQP_API_TOKEN")`

Reply:
If you have already done `save_account()`, then `QiskitRuntimeService()` is sufficient. 

For people who don't want to do `save_account()`, presumably they wouldn't want to save it in .bashrc either, since both store the credentials in plain text locally. In this case they'd have to manually code `QiskitRuntimeService(channel=..., token=..., instance=...)`. 

I wouldn't recommend doing `QiskitRuntimeService(token=os.environ.pop("IQP_API_TOKEN"))` because `save_account` saves more than just the token. And the code doesn't support mixing the profile info with parameter info. For example: 

```python
# This saves to the profile, including the instance
QiskitRuntimeService.save_account(channel=..., token=..., instance="h1/g1/p1") 

# This loads from the profile, including the instance
service = QiskitRuntimeService()
backend = service.backend("ibm_fez")
Sampler(backend).run(...)  #  <-- this will run using the instance h1/g1/p1

# This does NOT load from the profile and hence not the instance
service = QiskitRuntimeService(token=os.environ.pop("IQP_API_TOKEN"))
backend = service.backend("ibm_fez")
Sampler(backend).run(...)  #  <-- this will use a random instance
```